docs/runbooks/CDIDefaultStorageClassDegraded.md

```diff
@@ -0,0 +1,47 @@
+# CDIDefaultStorageClassDegraded
+<!-- Edited by agilboa, 4 Dec 2023 -->
+
+## Meaning
+
+This alert fires when the default (Kubernetes or virtualization) storage class supports smart clone (either CSI or snapshot based) and ReadWriteMany.
```

Comment:
@arnongilboa maybe a link about smart clone would be helpful. Like https://github.com/kubevirt/containerized-data-importer/blob/main/doc/smart-clone.md

Reply:
The link is in the Diagnosis section.